mozci/console/commands/push.py

```diff
@@ -314,7 +342,25 @@ def handle(self) -> None:
         except ValueError:
             raise Exception("Provided --backfill-limit should be an int.")
 
-        for push in pushes:
+        for push in self.pushes:
+            if self.option("disable-build-regressions"):
+                self.line("Skipping build regressions analysis")
+            else:
+                try:
+                    tasks_to_retrigger = check_build_regressions(push)
+                    if not tasks_to_retrigger:
+                        self.line("No build task detected as potential regression")
+                    else:
+                        self.line(
+                            f"Retriggering {len(tasks_to_retrigger)} build regression that may introduce a build bustage"
+                        )
+                        for task in tasks_to_retrigger:
+                            task.retrigger(push)
+                except Exception as e:
+                    self.line(
+                        f"<error> Couldn't run build regressions analysis on push {push.push_uuid}: {e}.</error>"
+                    )
```

Comment:
As a follow-up, we could later just merge this into `classify_regressions` and add to the `to_retrigger_or_backfill` list

Reply:
I opened PR #1158.
We should be able to rebase once we are sure it works with the try-catch behavior and the option to disable it.